cmd/gen/dependabot/flag.go

```diff
@@ -0,0 +1,13 @@
+package dependabot
+
+import (
+	"github.com/spf13/cobra"
+)
+
+type flag struct {
+	Reviewers string
+}
+
+func (f *flag) Init(cmd *cobra.Command) {
+	cmd.Flags().StringVar(&f.Reviewers, "reviewers", "", "Reviewers you want to assign automatically when Dependabot creates a PR, comma separated.")
```

Comment:
```suggestion
	cmd.Flags().StringVar(&f.Reviewers, "reviewers", "", "Reviewers you want to assign automatically when Dependabot creates a PR, comma separated, e.g. giantswarm/team-firecracker.")
```

Reply:
Let's also use `StringSliceVar`:

```suggestion
	cmd.Flags().StringSliceVar(&f.Reviewers, "reviewers", "", "Reviewers you want to assign automatically when Dependabot creates a PR, comma separated, e.g. giantswarm/team-firecracker.")
```